redash/destinations/hipchat.py

```diff
@@ -42,7 +42,7 @@ def notify(self, alert, query, user, new_state, app, host, options):
                 query_url=query_url)
 
             data = {
-                'message': message,
+                'message': message.encode('utf-8', 'ignore'),
```

Comment:
JSON is unicode, so wouldn't it be enough to construct a unicode string on [line 39](https://github.com/toyama0919/redash/blob/ae3af64c0966f95ef087d0be1741e5d9bb1de32b/redash/destinations/hipchat.py#L39)? I.e. change `message = "<a href..."` to `message = u"<a href..."`?